packages/controller/src/resources/monitoring/system/dashboards/cortex-overview-per-tenant.json

```diff
@@ -643,7 +643,7 @@
       "steppedLine": true,
       "targets": [
         {
-          "expr": "sum by (status_code) (rate(cortex_api_proxy_request_duration_seconds_count{job=\"$tenant-tenant.cortex-api\",method=\"GET\",route=~\"api_prom_api_v1_query.*\"}[$timewindow]))",
+          "expr": "sum by (status_code) (rate(cortex_api_proxy_request_duration_seconds_count{job=\"$tenant-tenant.cortex-api\",method=\"GET\",route=~\"metrics\"}[$timewindow]))",
```

Comment:
The only route available on the read path appears to be 'metrics'.  Is this expected @sreis @jgehrcke?

Reply:
The Grafana/browser based query path is expected to be a blind spot when using the data API proxy-based metrics! (those queries do not go through the data API proxies).

Reply:
Not quite following, are you saying that the `metrics` route is for browser-based queries?

Stepping back to the problem:  this graph appears empty without a change.  The only route that appears to be available is "metrics".